demo_scripts/train_model_arr.sh

```diff
@@ -16,18 +16,44 @@ set +a
 source ${MINICONDA_PATH}
 conda activate ${TOWERLLM_ENV_NAME}
 
-export CUDA_DEVICE_MAX_CONNECTIONS=1
-
-# Usage: sbatch demo_scripts/train_model.sh <checkpoint_path> <model config (see ./configs)> <dataset_bin> <external_tokenizer>
-# to use the wandb logger: --wandb_logger --wandb_entity <your username> --wandb_id <some id> --wandb_api_key <your api key>
 set -euo pipefail
 
 EXP_CONFIG=$1
+IFS=',' read -r task_id model_size nl_code_mix CHECKPOINT_PATH model_config data_mix_file external_tokenizer TOTAL_TRAIN_TOKENS cmd < <(sed "${SLURM_ARRAY_TASK_ID}q;d" $EXP_CONFIG)
```

Comment:
This reads the first line of the CSV which just contains column names - fixing this in commit